src/integrations/getting-started.md

```diff
@@ -0,0 +1,32 @@
+# Integrations Getting Started
+
+Unstoppable provides multiple ways to integrate. Unstoppable recommends preferred ways for each type of integration.
+
+![](../.gitbook/assets/integrations_decision_framework.svg)
+
+## References
+
+### Libraries
+* [JavaScript Resolution library](https://github.com/unstoppabledomains/resolution)
+* [Java Resolution library](https://github.com/unstoppabledomains/resolution-java)
+* [Swift Resolution library](https://github.com/unstoppabledomains/resolution-swift)
+
+### Website API documentation
+* [Website API reference](https://apidocs.unstoppabledomains.com/)
+
+### Read from blockchain
+Direct read from Unstoppable Smart Contracts without using Unstoppable libraries. Prior knowledge of blockchain is required to implement this method.       
+* [Smart contracts reference](../domain-registry-essentials/cns-smart-contracts.md)  
+* [Records reference](../domain-registry-essentials/records-reference.md)  
+* [Resolving domain records](../domain-registry-essentials/resolving-domain-records.md)  
+
+### Write to blockchain
+Direct write from Unstoppable Smart Contracts without using Unstoppable libraries. Prior knowledge of blockchain is required to implement this method.   
```

Comment:
```suggestion
Make writes directly to Unstoppable smart contracts. Prior knowledge of blockchain is required to implement this method.   
```

I wouldn't mention libraries for now, because we don't even have any for management.